packages/zkdb/src/sdk/authentication/authentication.ts

```diff
@@ -1,24 +1,24 @@
-import { AppContainer } from 'src/container.js';
+import { IApiClient } from '@zkdb/api';
 import storage from '../../storage/storage.js';
 import { SignedData } from '../../types/signing.js';
 import { Signer } from '../signer/interface/signer.js';
 import { ZKDatabaseUser } from '../types/zkdatabase-user.js';
 
 export class Authenticator {
   private signer: Signer;
```

Comment:
```suggestion
  #signer: Signer;
```
This one much more secure than `private`